src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1171,7 +1176,8 @@ private DetectProperties() {
     public static final DetectProperty<LongProperty> DETECT_TIMEOUT =
         new DetectProperty<>(new LongProperty("detect.timeout", 300L))
             .setInfo("Detect Timeout", DetectPropertyFromVersion.VERSION_6_8_0)
-            .setHelp("The amount of time in seconds Detect will wait for network connection, for scans to finish, and to generate reports (i.e. risk and policy check). When changing this value, keep in mind the checking of policies might have to wait for scans to process which can take some time.")
+            .setHelp(
```

Comment:
What happened here?

Reply:
Looks like IDE autocorrection based on the line length, I'll see what I can do to fix that.